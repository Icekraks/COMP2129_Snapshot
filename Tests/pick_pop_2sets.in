set a 1 2 3 4 5
get a
pick 2
get a
pop a
get a
set b 3 4 5 6
pop b
get b
list keys
bye
