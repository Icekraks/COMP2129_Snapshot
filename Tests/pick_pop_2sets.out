ok
[1 2 3 4 5]
2
[1 2 3 4 5]
1
[2 3 4 5]
ok
3
[4 5 6]
b
a
bye
